pkg/auto/validate.go

```diff
@@ -323,5 +324,14 @@ func loadAndCheckIconPrefix(ctx context.Context, rootFs billy.Filesystem, chart
 		return errors.New("icon path is not a file:// prefix, after make prepare, you need to run make icon for chart:" + chart)
 	}
 
+	exist, err := filesystem.PathExists(ctx, rootFs, strings.TrimPrefix(iconField, "file://"))
+	if err != nil {
+		return err
+	}
+
+	if !exist {
+		return errors.New("icon path is a file:// prefix, but the icon does not exist, after make prepare, you need to run make icon for chart:" + chart)
```

Comment:
What do you think about adding single quotes to the `make` commands? 
```suggestion
		return errors.New("icon path is a file:// prefix, but the icon does not exist, after 'make prepare', you need to run 'make icon' for chart:" + chart)
```